### DISCUSSION TITLE
Cannot export unpublished changes to a branch

### DISCUSSION THREAD

**Discussion by SomePogProgrammer** [community/community#66698] [Codespaces] [answered]
### Select Topic Area

Show & Tell

### Body

When I reached the 120-hour core limit for codespaces, I was in the middle of trying to push a fix to my GitHub repository, and I was unable to publish the changes. So I thought it'd be okay because I can export the changes to a branch, but whenever I try that, it just loads for a while, giving me an error. 
_**_Any possible way I can fix this issue? Or export the unpublished changes to a branch or use another way?_**_

![image](https://github.com/community/community/assets/92398408/d088634b-2c25-4fda-89d8-32e762b00da5)

I appreciate any help, thanks!

**Comment by SomePogProgrammer** [comment]
I'm still experiencing the same issue, if anyone is able to give me insight on why this is or help me fix it, I'd appreciate it!

**Reply by ghost** [reply, marked as answer]
This happens with me too.

## What you can do
You can download the data of the codespace locally. Run this command on your local machine:
- **Syntax**
```bash
gh cs cp -e -r "remote:/workspaces/<codespace name>" <directory>
```
- **Example**
```bash
gh cs cp -e -r "remote:/workspaces/codespaces-blank" ./Repo
```
After this you can publish your work to a repository
> [!NOTE]
> Make sure that you have installed the [`GitHub CLI`](https://cli.github.com/)

## OR
You can just wait for the github team to resolve this issue

**Reply by SomePogProgrammer** [reply]
Will do, thanks for the response. I appreciate it!

**Reply by ayan479** [reply]
<img width="805" alt="Screen Shot 2024-05-11 at 4 42 50 PM" src="https://github.com/community/community/assets/108748711/83f647ee-c0b4-476e-95d9-e10b5ef8fe88">
Doesnt seem to work.

**Reply by bryantamayo1** [reply]
I have the same problem, in my case I spent 100% of GitHub Pro and I can't create a branch with unpushed changes, web page show me this message:
"Your codespace export did not complete. Please try again."
